src/main/java/core/basesyntax/service/service/impl/FileParserImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.service.impl;
+
+import core.basesyntax.model.FruitModel;
+import core.basesyntax.model.OperationModel;
+import core.basesyntax.service.FileParserService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileParserImpl implements FileParserService {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_TYPE_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<OperationModel> parse(List<String> listFromFile) {
+        List<OperationModel> transactionList = new ArrayList<>();
+        for (int i = 1; i < listFromFile.size(); i++) {
```

Comment:
```suggestion
        for (int i = OFFSET; i < listFromFile.size(); i++) {
```
let's make it more visible, it can be easily overlooked

Reply:
Not fixed